### PR TITLE
96 add filtering by interval to selection method

### PR DESCRIPTION
**Pull request by matdenkas**

### CODE REVIEW

File: src/SeriesStorage/SeriesStorage/SQLAlchemyORM.py
Comment (bestrada33) [inline src/SeriesStorage/SeriesStorage/SQLAlchemyORM.py:1]: Need documentation for `timeIntervalConstraint`

File: src/SeriesStorage/SeriesStorage/SQLAlchemyORM.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def __init__(self) -> None:
     ################################################################################## Public methods
     #############################################################################################
 
-    def select_input(self, seriesDescription: SeriesDescription, timeDescription : TimeDescription) -> Series:
+    def select_input(self, seriesDescription: SeriesDescription, timeDescription : TimeDescription, timeIntervalConstraint: timedelta = None) -> Series:
```
Comment (bestrada33) [inline src/SeriesStorage/SeriesStorage/SQLAlchemyORM.py:42]: Need documentation for `timeIntervalConstraint`

**Review by bestrada33 — CHANGES_REQUESTED** [review]
Need documentation for `timeIntervalConstraint`

**Review by bestrada33 — APPROVED** [review]
Test looks great. Small errors fixed. LGTM

**Review by lovelysandlonelys — CHANGES_REQUESTED** [review]
When running the test files two tests fail for the same reason: 
image

**Comment by matdenkas** [comment]
> When running the test files two tests fail for the same reason: image

image

**Review by bestrada33 — APPROVED** [review]
LGTM again

**Pull request merged**